Doc/library/unittest.rst

```diff
@@ -1278,6 +1310,31 @@ Test cases
 
       .. versionadded:: 3.2
 
+   .. method:: assertStartsWith(s, prefix, msg=None)
+   .. method:: assertNotStartsWith(s, prefix, msg=None)
+
+      Test that the unicode or byte string *s* starts (or does not start)
```

Comment:
As suggested in the style guide: https://devguide.python.org/documentation/style-guide/#specific-words
```suggestion
      Test that the Unicode or byte string *s* starts (or does not start)
```